src/network/MissionControlProtocol.cpp

```diff
@@ -452,7 +472,8 @@ void MissionControlProtocol::updateArmIKRepeatTask() {
 			robot::getMotorPositionsRad(Constants::arm::IK_MOTORS);
 		if (armJointPositions.isValid()) {
 			navtypes::Vectord<Constants::arm::IK_MOTORS.size()> targetJointPositions =
-				Globals::planarArmController.getCommand(dataclock::now(), armJointPositions);
+				Globals::planarArmController.value().getCommand(dataclock::now(),
+																armJointPositions);
```

Comment:
We can replace `planarArmController.value().foo` with `planarArmController->foo`, which is more concise, so we should do that everywhere